### PR TITLE
Fixing Issue 1097 Incorrect comment in kubeadm

### PR DESCRIPTION
**Pull request by rakrup**
## Fixing Issue 1097 Incorrect comment in kubeadm

- the type of fix -  bug fix 
- some details on _why_ this PR should be merged : Fixes issue #1097 
- This fixes the comment in the kudeadm manifest.

### CODE REVIEW

File: master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml [hunk 1]
```diff
@@ -67,14 +67,15 @@ spec:
         # a failure.  This annotation works in tandem with the toleration below.
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
-      # Only run this pod on the master.
+      # Toleration allows the pod to run on master
```
Comment (tmjd) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml]: Could you move the comment to just above the `key: node-role.kubernetes.io/master` so it doesn't sound like the CriticalAddonsOnly one is for the same thing?
Also please add this in the same place down below for the calico-node DaemonSet as it is appropriate there too.

**Comment by caseydavenport** [comment]
@rakrup ping, any word on this guy? I'd like to merge it once the last comment is addressed!

**Comment by rakrup** [comment]
@caseydavenport @tmjd . Hey, just fixed the review comments. Please merge if they look fine. Sorry to keep you folks waiting on this one.

**Comment by tmjd** [comment]
LGTM

**Comment by ozdanborne** [comment]
@rakrup just checking in. If you could please rebase, this is ready to go in :smile:

**Comment by rakrup** [comment]
Hey Dan, I am out for a festive vacation at a remote location with extremely limited connectivity to internet. I would be back home by Monday and doing this would be on top priority on my task list 🙂

**Comment by caseydavenport** [comment]
@rakrup I think this just needs a rebase, but note that the 1.6/calico.yaml file has been moved to 1.7/calico.yaml, so you'll need to copy the changes over there instead.

**Comment by caseydavenport** [comment]
I've finished this one off here: https://github.com/projectcalico/calico/pull/1792

**Pull request closed without merging**